src/api/nodeRpcCall.js

```diff
@@ -1104,61 +1104,24 @@ const getOfficialUserRegistryEntries = async (walletAddress) => {
   }
   let companyRequestsByWallet = []
   let registeredCompaniesByWallet = []
-  let supportedObject = JSON.parse(JSON.stringify(newCompanyDataObject))
   companyRegistryRawData.forEach((companyRegistryEntity, index) => {
     let companyData;
     try {
       const compressed = companyRegistryEntity.unwrap().data;
-      companyData = api.createType('CompanyData', pako.inflate(compressed)).toJSON();
+      companyData = api.createType('CompanyData', pako.inflate(compressed));
     } catch(e) {
       console.error("Invalid company data", e); // FIXME actually display this to user somehow?
       return;
     }
 
-    const staticFields = []
-    const dynamicFields = []
+    // FIXME this is component-specific logic, nodeRpcCall shouldn't do this
```

Comment:
Do the fixme? Or if its done remove it

Reply:
It's a risky, multi-day refactor that's not really relevant to what this PR is about. It's something to be done when there's a bit more time.